xml/cap_admin_app_domains.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.cap.custom-app-domains"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+<!-- TODO review if chapter is still required. See:
+  https://github.com/SUSE/doc-cap/issues/283
+  -->
+  <title>Custom Application Domains</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <para>
+  In a standard &scf; deployment, applications will use the same domain as the
+  one configured in your <filename>scf-config-values.yaml</filename> for SCF.
+  For example, if your &scf; deployment's domain is set as
+  <literal>example.com</literal> then an application's domain will be in the
+  form of <literal>myapp.example.com</literal>.
+ </para>
```

Comment:
Does the new app domain need a DNS entry?

Reply:
?

Reply:
@andreas-kupries Do you know if a DNS entry is needed when doing this?

Reply:
I suspect so. Note, the original information I recorded in the wiki page came from @jandubois
Ping.

Reply:
I don't understand the question. There must be some way to resolve the domain name, otherwise how can you route to it? So you need it either in DNS, or be a name supported by mDNS, or have it in your local `/etc/hosts`, or whatever else your users will use to resolve names to IP addresses.

This kind of feels obvious, so I suspect I somehow managed to miss the real point of the question. Please help!

Reply:
@jandubois Nothing is obvious with CAP and the underlying stack, that is why we ask!

Reply:
@cjschroder Then please rephrase your question. So far I read: "Will the URL work without a DNS entry?", and this has nothing to do with CAP. How would you expect the client to resolve the address if there is no entry for it?

But if this is indeed your question, then the answer is: "Yes, every domain needs a DNS entry to be resolvable. Either a wildcard entry for the application domain, or individual entries for each app."

Reply:
BTW, I just noticed a naming inconsistency in this doc part: What you call "deployment domain" is called "application domain" in Cloud Foundry terminology. And what you call "application domain" would just be the "application URL". I think we should stick to upstream terminology here.